core/bones/json.py

```diff
@@ -0,0 +1,54 @@
+import json
+from viur.core.bones import BaseBone
+
+
+class JsonBone(BaseBone):
+    """
+    This bone saves its content as a JSON-string.
+    """
+
+    type = "json"
+
+    def __init__(self, indexed=False, multiple=False, languages=None, *args, **kwargs):
+        assert not multiple
+        assert not languages
+        assert not indexed
+        super().__init__(*args, **kwargs)
+
+    def serialize(self, skel, name, parentIndexed):
+        if name not in skel.accessedValues:
+            return False
+
+        value = skel.accessedValues[name]
+        if isinstance(value, str):
+            try:
+                _ = json.loads(value)
+                del _
+            except Exception as e:
+                raise f"Error in serialize in JsonBone: {e=}"
```

Comment:
You cannot raise a str. But catching an exception and raising another doesn't make sense to me here.

Reply:
Please raise a ValueError